app/Http/Controllers/GroupsController.php

```diff
@@ -43,9 +43,12 @@ public function create(Request $request) : View
         $permissions = config('permissions');
         $groupPermissions = Helper::selectedPermissionsArray($permissions, $permissions);
         $selectedPermissions = $request->old('permissions', $groupPermissions);
-
+        $users = \App\Models\User::get();
```

Comment:
Couldn't this be a problem if we had, say, 30,000 users (which does happen for some larger users/customers)?

And if we wanted to still do this I would prefer to use `::all()` because it's clearer that you're grabbing _everything_.

Reply:
Yep. But this is what I had to do to get it working for now.